cfgov/legacy/tests/views/test_housing_counselor.py

```diff
@@ -11,13 +11,13 @@ class HousingCounselorS3URLMixinTestCase(TestCase):
     def test_s3_json_url(self):
         self.assertEqual(
             HousingCounselorS3URLMixin.s3_json_url(20001),
-            'https://s3.amazonaws.com/foo.bucket/a/assets/hud/jsons/20001.json'
```

Comment:
It (and the other changes you call out) do accurately reflect the logic change here: https://github.com/cfpb/cfgov-refresh/pull/4596/files/cad4004a5a558db5ba2f12c19b192274946bc7c9#diff-2e9849ee42dc1b061a792d466e8a0d6bR4   (the point of which, is so that https_s3_url_prefix returns https://files.consumerfinance.gov instead of https://s3.amazonaws.com/files.consumerfinance.gov )

I did have a late-breaking realization that the URL's generated there don't seem to actually get sent to the browser, it's all used by server-side code, so it might not matter that much. I'll think about this a little more.

Reply:
( and just to be clear, the reason it's 'foo.bucket' in these tests, not files.consumerfinance.gov is the [override_settings](https://github.com/cfpb/cfgov-refresh/blob/cad4004a5a558db5ba2f12c19b192274946bc7c9/cfgov/legacy/tests/views/test_housing_counselor.py#L9) blocks)